chat-sdk-core/src/main/java/co/chatsdk/core/interfaces/InterfaceAdapter.java

```diff
@@ -45,6 +46,14 @@
 
     Activity profileActivity (User user);
 
+    void addTab(Tab tab);
+    void addTab(int index, Tab tab);
```

Comment:
This should be changed to `void addTab(Tab tab, int index)` always try to keep the order of the arguments consistent. Since the first one is `void addTab(Tab tab)`, it follows Tab will always be the first argument.